src/audio/tone.c

```diff
@@ -143,7 +143,8 @@ static void tone_s32_default(struct comp_dev *dev, struct comp_buffer *sink,
 				dest++;
 			}
 		}
-		tone_circ_inc_wrap(&dest, sink->end_addr, sink->size);
+		tone_circ_inc_wrap(&dest, sink->end_addr,
```

Comment:
Checkpatch (line length) fix should be in a separate commit, although multiple such fixes can be brought together?

Reply:
It's leftover renaming, good catch